### DISCUSSION TITLE
Cannot activate G90 style

### DISCUSSION THREAD

**Discussion by jgperrin** [carbon-design-system/carbon-charts#1140] [Q&A] [answered]
My whole app is based on G90 style, but I keep getting the g0 one:

Working screen:

![Screen Shot 2021-09-09 at 15 34 35](https://user-images.githubusercontent.com/6397413/132751212-ee41606d-f7cd-452b-9616-09238e6688ec.png)

Not so working screen:

![Screen Shot 2021-09-09 at 15 34 45](https://user-images.githubusercontent.com/6397413/132751243-98258e35-e1a8-401d-8a03-099d373b9e8f.png)

I have all my styling in index.scss:

```
@import "@carbon/themes/scss/themes";
@import "@carbon/charts/styles.css";

$carbon--theme: $carbon--theme--g90;
@include carbon--theme();

@import "carbon-components/scss/globals/scss/styles.scss";

```
I tried to move things around, but I probably missed a thing or two :(...

**Comment by theiliad** [comment, marked as answer]
Hi, this is how our demo site does it

https://github.com/carbon-design-system/carbon-charts/blob/0309a546b1dfcd4509a9c55a77457e6515bddd09/packages/core/demo/styles.scss#L64-L65

https://github.com/carbon-design-system/carbon-charts/blob/0309a546b1dfcd4509a9c55a77457e6515bddd09/packages/core/src/styles/styles-g90.scss#L1-L5
